Shared/SentryIntegration.swift

```diff
@@ -54,44 +53,44 @@ public class Sentry {
             return
         }
 
-        var sentryDSNKey = SentryDSNKey
+        var environment = "Production"
         if AppInfo.appVersion == AppConstants.NIGHTLY_APP_VERSION, AppConstants.BuildChannel == .beta {
             // Setup sentry for Nightly Firefox Beta
-            sentryDSNKey = SentryNightlyDSNKey
+            environment = "Nightly"
```

Comment:
nit: would be nice to make them enum or static vars so we can easily change the env